examples/webgl_materials_envmaps_hdr.html

```diff
@@ -146,7 +144,7 @@
 				};
 
 				var hdrUrls = genCubeUrls( "../examples/textures/cube/pisaHDR/", ".hdr" );
-				new THREE.HDRCubeTextureLoader().load( hdrType, hdrUrls, function ( hdrCubeMap ) {
+				new THREE.HDRCubeTextureLoader().load( THREE.UnsignedByteType, hdrUrls, function ( hdrCubeMap ) {
```

Comment:
FloatType worked for me on my Windows PC.

I believe it is related to asking for the right extension before using it.  You need both OES_texture_float and oes_texture_float_linear available for float, and oes_texture_half float_oes_texture_half_float_linear for Half support.